TeamCode/src/main/java/org/firstinspires/ftc/teamcode/core/lib/autonomousControl/RobotMovementState.java

```diff
@@ -0,0 +1,31 @@
+package org.firstinspires.ftc.teamcode.core.lib.autonomousControl;
+
+import org.firstinspires.ftc.teamcode.robot.constants.AutonomousConstants;
+
+public class RobotMovementState {
+    //i think this whole class could be eliminated, but i have to develop
```

Comment:
do you have updates on this comment?

Reply:
Yes, this class is actually useless when it comes to webcam automation's functionality. It is only there because the whole Webcam branch has it's roots in a more advanced control system that would have used said class. Consider it removed (The same applies to TargetVelocityData now that I checked it)